eis_qgis_plugin/processing/algorithms/exploratory_analysis/local_morans_i.py

```diff
@@ -0,0 +1,80 @@
+from qgis.core import (
+    QgsProcessingParameterEnum,
+    QgsProcessingParameterFeatureSource,
+    QgsProcessingParameterField,
+    QgsProcessingParameterNumber,
+    QgsProcessingParameterVectorDestination,
+)
+
+from eis_qgis_plugin.processing.eis_processing_algorithm import EISProcessingAlgorithm
+
+
+class EISLocalMoransI(EISProcessingAlgorithm):
+    def __init__(self) -> None:
+        super().__init__()
+
+        self._name = "local_morans_i"
+        self._display_name = "Local Moran's I"
+        self._group = "Exploratory analysis"
+        self._group_id = "exploratory_analysis"
+        self._short_help_string = (
+            "Perform Local Moran's I calculation."
+        )
+
+    def initAlgorithm(self, config=None):
+        self.alg_parameters = [
+            "input_vector",
+            "output_vector",
+            "column",
+            "weight_type",
+            "k",
+            "permutations",
+        ]
+
+        input_vector_param = QgsProcessingParameterFeatureSource(
+            name=self.alg_parameters[0],
+            description="Input geometries"
+        )
+        input_vector_param.setHelp("Input geometries that contains the data to be examined with Local Moran's I.")
```

Comment:
A small misspell here: change "contains" to "contain" because "geometries" is plural